### PR TITLE
[TS] LPS-116940 Fix wrong visualization when rendering inline SVG markup

### PR DESCRIPTION
**Pull request by dacousalr**
Hi team,

This is a proposal for LPS-116940. More details can be found in [PTR-1822](https://issues.liferay.com/browse/PTR-1822)

Thanks in advance 

\cc @antonio-ortega

### CODE REVIEW

File: portal-web/docroot/html/common/themes/bottom_js.jspf [hunk 1]
```diff
@@ -64,7 +64,14 @@
...
 			{
 				attributeName: 'data-href',
-				polyfill: true
+				polyfill: true,
+				validate: function (src, svg, use) {
+					if (src && src.startsWith('#')) {
+						return false;
+					}
+
+					return true;
```
Comment (wincent) [inline portal-web/docroot/html/common/themes/bottom_js.jspf:73]: Probably best written as a single `return`:

```js
return !(src && src.startsWith('#'));
```

or:

```js
return !src || !src.startsWith('#');
```

**Comment by wincent** [comment]
Just pinging on @jbalsas and @dsanz, seeing as you both commented on @antonio-ortega's Slack thread. I understand from your comments that you're both fine with this workaround, so speak now or forever hold your peace!

I'm going to run tests in the meantime to see if we can get a clean run (not that I expect CI to catch any potential problems with this...)

The only other thing I wanted to check is that our IE polyfill has run by the time we make that `startsWith()` call — I believe it does, as it's [going in to `top_head.jsp`](https://github.com/liferay/liferay-portal/blob/fcffc8af664fd67a6f1fd01a460824265f35349a/modules/apps/frontend-compatibility/frontend-compatibility-ie/src/main/java/com/liferay/frontend/compatibility/ie/internal/servlet/taglib/IETopHeadDynamicInclude.java#L85) and the `startsWith()` call is happening in `bottom_js.jspf`.

**Comment by wincent** [comment]
@dacousalr: I pushed one extra commit ([778ecdfd72](https://github.com/liferay-frontend/liferay-portal/pull/104/commits/778ecdfd72f3bd7d7e681c7e6c4c8f2efb14afc1)) with the suggested simplication.

**Comment by wincent** [comment]
ci:test:sf

**Comment by wincent** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
This is green and @pat270 confirmed in the [JIRA ticket](https://issues.liferay.com/browse/LPS-116940) that the svg is indeed valid. Forwarding since @wincent already approved and pushed the changes.

**Comment by jbalsas** [comment]
ci:forward

**Pull request closed without merging**